libs/core-ui/src/lib/Highchart/FeatureImportanceBar.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import _ from "lodash";
+import { getTheme, Text } from "office-ui-fabric-react";
+import React from "react";
+
+import { getFeatureImportanceBarOptions } from "../util/getFeatureImportanceBarOptions";
+import { getFeatureImportanceBoxOptions } from "../util/getFeatureImportanceBoxOptions";
+import { ChartTypes } from "../util/IGenericChartProps";
+import { JointDataset } from "../util/JointDataset";
+
+import { BasicHighChart } from "./BasicHighChart";
+import { featureImportanceBarStyles } from "./FeatureImportanceBar.styles";
+
+export interface IGlobalSeries {
+  unsortedAggregateY: number[];
+  // feature x row, given how lookup is done
+  unsortedIndividualY?: number[][];
+  unsortedFeatureValues?: number[];
+  name: string;
+  colorIndex: number;
+  id?: number;
+}
+
+export interface IFeatureBarProps {
+  jointDataset: JointDataset | undefined;
+  chartType: ChartTypes;
+  yAxisLabels: string[];
+  sortArray: number[];
+  selectedFeatureIndex?: number;
+  selectedSeriesIndex?: number;
+  topK: number;
+  unsortedX: string[];
+  unsortedSeries: IGlobalSeries[];
+  originX?: string[];
+  xMapping?: string[];
+  onFeatureSelection?: (seriesIndex: number, featureIndex: number) => void;
+}
+
+export class FeatureImportanceBar extends React.Component<IFeatureBarProps> {
```

Comment:
can you add a screenshot of what this new component will look like?

Reply:
updated